src/main/java/org/killbill/billing/plugin/adyen/client/jaxws/LoggingOutInterceptor.java

```diff
@@ -34,7 +34,8 @@ public LoggingOutInterceptor(final Iterable<Obfuscator> obfuscators) {
     protected String transform(final String originalLogString) {
         String result = originalLogString.replaceAll("cvc>[0-9]+</", "cvc>***</")
                                          .replaceAll("number>[0-9]+</", "number>***</")
-                                         .replaceAll("bankAccountNumber>[0-9]+</", "bankAccountNumber>***</");
+                                         .replaceAll("bankAccountNumber>[0-9]+</", "bankAccountNumber>***</")
+                                         .replaceAll("Authorization=\\[[^\\]]*\\]", "Authorization=[****]");
```

Comment:
👍 

Nit: might make sense to enhance the [PatternObfuscator](https://github.com/killbill/killbill/blob/73df2253ad000c3596193983928cc3a449184770/profiles/killbill/src/main/java/org/killbill/billing/server/log/obfuscators/PatternObfuscator.java#L27) (which we could make it configurable more easily...).